client/my-sites/customer-home/cards/notices/celebrate-notice/index.jsx

```diff
@@ -44,12 +46,30 @@ const CelebrateNotice = ( {
 	const showNextTask = () => {
 		setIsLoading( true );
 		dispatch( skipCurrentViewHomeLayout( siteId ) );
+
+		dispatch(
+			composeAnalytics(
+				recordTracksEvent( 'calypso_customer_home_notice_show_next', {
```

Comment:
Am I understanding the code right in that this click for the main CTA could be for either "Show me what's next" or "Show site setup".

where "Show me what's next" is for non-site check list / post-launch items.   

If that is the case, then we probably want to separate the tracking, ya?     


![image](https://user-images.githubusercontent.com/5665959/119600056-8e950700-be29-11eb-9078-fa9f17098ba2.png)

![image](https://user-images.githubusercontent.com/5665959/119600065-95237e80-be29-11eb-8209-c9dc24b23c2f.png)

Reply:
Ohhhh. I think I have answered my own question.   The `is_site_setup_complete` prop provides a way to differentiate!

To avoid the confusion I just experienced, I'd use consistent tracking terminology (it seems to me you've mixed up Button language from the above two cards).  Better might be:

calypso_customer_home_notice_show_next 
calypso_customer_home_notice_dismiss

OR
calypso_customer_home_notice_show_site_setup 
calypso_customer_home_notice_skip

OR maybe best is
calypso_customer_home_notice_sitelaunch_show
calypso_customer_home_notice_sitelaunch_skip

Reply:
It wasn't clear in my original description (I've added a small note now) but this change actually adds tracks events for all the My Home notices, not just the post-launch one. That's why the event is `*_show_next`, because that's a generic term that covers the primary CTA for all notices.

So the event props for the "Show site setup" event will look like:
```json
{
  "_en": "calypso_customer_home_notice_show_next",
  "notice": "home-notice-celebrate-site-launch",
  "is_site_setup_complete": false
}
```

And the event props for "Show me what's next" will look like:
```json
{
  "_en": "calypso_customer_home_notice_show_next",
  "notice": "home-notice-celebrate-site-launch",
  "is_site_setup_complete": true
}
```

I like the `*_dismiss` name for the skip button, but I was trying to keep the event names consistent with those sent by the "task" cards: `calypso_customer_home_task_start` and `calypso_customer_home_task_skip`

Aside: I do not know what the practical/design/intent difference is between "Notices" and "Tasks"

Reply:
Ah ha! Got it.  That makes sense. ![bitmoji](https://sdk.bitmoji.com/render/panel/469d4aea-0aaa-4b89-919e-0fd853b6f0f0-b931236c-c79a-49b5-b0f5-ab26d7e4cd09-v1.png?transparent=1&palette=1&width=40)